From 4348c72877e049804c7cdb7eec7a6bcd7ac2ea1d Mon Sep 17 00:00:00 2001
From: Bartosz Golaszewski <brgl@bgdev.pl>
Date: Wed, 11 May 2022 13:21:30 +0200
Subject: [PATCH] setup: don't use setuptools-markdown

This project is deprecated and irrelevant for the functionality of
eth-keyfile. We don't support it in meta-python so just drop it from
the dependencies.

Signed-off-by: Bartosz Golaszewski <brgl@bgdev.pl>
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index cbb7f4c..7702aa4 100644
--- a/setup.py
+++ b/setup.py
@@ -51,14 +51,12 @@ setup(
         "A library for handling the encrypted keyfiles used to store ethereum "
         "private keys."
     ),
-    long_description_markdown_filename='README.md',
     author='Piper Merriam',
     author_email='pipermerriam@gmail.com',
     url='https://github.com/ethereum/eth-keyfile',
     include_package_data=True,
     install_requires=install_requires,
     extras_require=deps,
-    setup_requires=['setuptools-markdown'],
     py_modules=['eth_keyfile'],
     license="MIT",
     zip_safe=False,
-- 
2.34.1

